widget/toolbar.go

```diff
@@ -13,31 +13,48 @@ type ToolbarItem interface {
 	ToolbarObject() fyne.CanvasObject
 }
 
+type toolbarActionButton struct {
```

Comment:
I think you may need to create a constructor for this and call `ExtendBaseWidget` in the constructor

Reply:
Thanks, that fixed my rendering issue! Added a commit.